### DISCUSSION TITLE
how to hide close icon in p-multiselect  and chnage dropdown menu item

### DISCUSSION THREAD

**Discussion by mdnafeed** [primefaces/community#69] [PrimeNG] [answered]
i need help how to hide close icon without writing css and change background color  

![image](https://github.com/primefaces/community/assets/17257897/9a446aea-0e9b-4df1-8e8a-f2bc5b697bcd)

**Comment by cetincakiroglu** [comment, marked as answer]
Hi,

Unfortunately, multiselect does not have any option to disable the close icon. You can work around it by using custom CSS or using filter template to make your custom filter section [see the example](https://stackblitz.com/edit/xwpcry?file=src%2Fapp%2Fdemo%2Fmulti-select-basic-demo.html). 

Also, you can request that feature by opening a feature request issue on the repo.

**Comment by mdnafeed** [comment]
Thank you for your update!!

**Comment by Rishikesh1027** [comment]
just pass "closeIcon" as prop
